projects/uswds-components/src/lib/form-field/form-field.component.ts

```diff
@@ -0,0 +1,112 @@
+import { Component, Input, ChangeDetectorRef } from '@angular/core';

```

Comment:
Overall, I like this wrapper idea for labels, descriptions, and errors. We've kept labels separate from form controls, so it's a nice feature to add for users if they want to use a pre-built component for also rendering the labels / descriptions / errors